website/docs/r/compute_cluster_vm_affinity_rule.html.markdown

```diff
@@ -48,37 +55,37 @@ will run on the same host whenever possible.
 [tf-vsphere-vm-resource]: /docs/providers/vsphere/r/virtual_machine.html
 
 ```hcl
-data "vsphere_datacenter" "dc" {
-  name = "dc1"
+data "vsphere_datacenter" "datacenter" {
+  name = "dc-01"
 }
 
 data "vsphere_datastore" "datastore" {
-  name          = "datastore1"
-  datacenter_id = "${data.vsphere_datacenter.dc.id}"
+  name          = "datastore-01"
+  datacenter_id = "data.vsphere_datacenter.datacenter".id"
```

Comment:
I think removing the double quotes were missed when switching from the old interpolation syntax to the new one